src/utils/logger.ts

```diff
@@ -13,20 +13,33 @@ const stringify = (obj: any) => {
 }
 
 // If you need logs during tests you can set the env var TEST_LOGGING=true
-const getLoggerType = () => {
+const getLoggerConfig = (): {
+  type: 'pretty' | 'json'
+  minLevel: number
+} => {
   if (process.env.NODE_ENV === 'development') {
-    return 'pretty'
+    return {
+      type: 'pretty',
+      minLevel: 0,
```

Comment:
We have a logging level in the .env file too, we should use that

Reply:
https://github.com/github-community-projects/private-mirrors/blob/cdb56dad989e749f800b55e3ed7b93b38c1fafbc/env.mjs#L20